spark/src/test/scala/org/apache/comet/CometCastSuite.scala

```diff
@@ -87,13 +87,11 @@ class CometCastSuite extends CometTestBase with AdaptiveSparkPlanHelper {
 
   // CAST from BooleanType
 
-  ignore("cast BooleanType to ByteType") {
-    // https://github.com/apache/datafusion-comet/issues/311
+  test("cast BooleanType to ByteType") {
     castTest(generateBools(), DataTypes.ByteType)
   }
```

Comment:
This is not supported before and supported now?

Reply:
Not quite. It was disabled before and now it is enabled. With this PR, I enabled all of the tests and then kept the ones that are passing.

Reply:
Got it. Thanks.